src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Debugging/DefaultRazorProximityExpressionResolver.cs

```diff
@@ -128,7 +102,13 @@ internal class DefaultRazorProximityExpressionResolver : RazorProximityExpressio
                 return null;
             }
 
-            var syntaxTree = await virtualDocument.GetCSharpSyntaxTreeAsync(_workspace, cancellationToken).ConfigureAwait(false);
+            if (!_workspaceAccessor.TryGetWorkspace(textBuffer, out var workspace))
+            {
+                // Couldn't find an associated workspace for the buffer
+                return null;
```

Comment:
Ought this be a logged error? If not can we include an explanation on what kind of scenario this might be?

Reply:
I don't know of a scenario when this would actually happen in practice but worth logging